GVFS/GVFS.Platform.Mac/ProcessLauncher.cs

```diff
@@ -37,7 +38,7 @@ public static void StartBackgroundProcess(string programName, string[] args)
 
                     if (processId == 0)

```

Comment:
Should we also log above (when `processId == -1`) before throwing a `Win32Exception` (similar to what's being done in WindowsPlatform.cs)?

Reply:
Makes sense.